### PR TITLE
UCS, UCT: Remove epoll keyword from comments.

### LINKED ISSUE #3900: [Q] epoll keyword in comments.
Hello, @dmitrygx 

As you know, we eliminate epoll function in many files.
Does this comment are still correct or should we fix it?

Best regards.

```C
src/ucs/async/async.h
56: * @param mode            Either to use signals or epoll threads to wait.

src/ucs/async/async_fwd.h
109: * @param mode            Either to use signals or epoll threads to wait.

src/uct/tcp/tcp.h
20:/* How many events to wait for in epoll_wait */
```

### PR DESCRIPTION
**Pull request by hiroyuki-sato**
## What

Remove `epoll` keyword from comments. Fix #3900 

## Why ?

We eliminate epoll function in files. Fix comments properly.

Could you review @dmitrygx ?

### CODE REVIEW

**Comment by yosefe** [comment]
ok to test

File: src/uct/tcp/tcp.h [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 #define UCT_TCP_NAME                "tcp"
 
-/* How many events to wait for in epoll_wait */
+/* Maximum number of times to poll on a ready socket. */
```
Comment (dmitrygx) [inline src/uct/tcp/tcp.h]: It sounds like a wrong statement
It should be something:
`Maximum number of events to wait from event set`

File: src/ucs/async/async.h [hunk 2]
```diff
@@ -53,7 +53,7 @@ void ucs_async_global_cleanup();
  * This can be used to ensure safe event delivery.
  *
  * @param async           Event context to initialize.
- * @param mode            Either to use signals or epoll threads to wait.
+ * @param mode            Either to use signals or poll threads to wait.
```
Comment (dmitrygx) [inline src/ucs/async/async.h]: poll -> polling

File: src/ucs/async/async_fwd.h [hunk 3]
```diff
@@ -106,7 +106,7 @@ ucs_status_t ucs_async_modify_handler(int fd, int events);
  * Allocate and initialize an asynchronous execution context.
  * This can be used to ensure safe event delivery.
  *
- * @param mode            Either to use signals or epoll threads to wait.
+ * @param mode            Either to use signals or poll threads to wait.
```
Comment (dmitrygx) [inline src/ucs/async/async_fwd.h]: poll -> polling

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/11332/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 33 workers (click for details)</summary>

**Note:** the logs will be deleted after 25-Jul-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/a05a0da07711949216880e36b1f307e5715ab44e/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/f71b3cee70d4b4c77e534b3af807f7ea55bf7658/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/3c879856218876fa136802e33eddbfdd32570975/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/e0383eaddabc0597d8b32fa36ccd8ccc89771acd/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/2caff4372018dabf2d6f925bd5c5beb2f6654475/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/6d4b8bccd66c9f4ef5eaed9a4ecd6ea91b2de67a/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/8e22c689f2306bd03c71abac9cf0330d8fc1a8d4/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/2c46d2b2c5965b47c4a904044b9937f79bec4e55/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/0efbe34593956a8bb3386800b71c3b8a767ef628/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/4171e4c5f661c1351a6a01c751801b9c1712d9ac/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/a03db3da8dd2a15e7f4f8e3abc612935a368215f/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/00f1a11c242e3fb56f15e67fce96dec1cf02ce63/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/0b53ca0e01f6c18706e9d6f8d63879fd4d836856/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/f144cbd42494d19885352d2aae35c5b11c0a0a95/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/29aa7bb0d2652b91e35d3371a3d6ee28c1ad8c1f/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/ea822d221673fb2fc3e82555d76e83a3fee81ef6/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/1f8b394b9458f7b708d2a7e487e53e9ffeef265c/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/af101921af6ea645b96576c131b33f512ad2b629/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/534d472298f878bd47743d28625ca6769bd6e631/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/b9d8cb38d77ac196caef0ef2c3913a494f881b1b/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/4de732637073e3ad0a8ef6aaa1dbafe969884bd5/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/8069d1e3de149b38eaea93ad10ab50e2ff14895e/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/67bfcd572d134aebc27e64cf375f3ba6e7a17221/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/30c3463c2fbd8a592582a677811f514c641e1a67/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/8ccafa828b0ee03d5547f3e6048b916a463963d6/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W0](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/94ba5670cb626638e7355a6939e866b2073812e7/hpc-test-node-upstream_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W1](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/0a26ef2cbb0f93b85f23246b3b3fbcb8ebf23193/hpc-test-node-upstream_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W2](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/8e2b3ba11a1cd2066f735570879e486594cbdfb7/hpc-test-node-upstream_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W3](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/83629393d25dbd5ea9bbc54e18a415fddba54e75/hpc-test-node-upstream_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/249a3e3b3170c7b4f105bd9cbd1520f52a157f80/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/e2e63c3d7bf3e2eeb3024fe58be859498d1e30d4/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/0acd9b935e398707c27400e56e97680e590b58c3/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/f5ce9d1964640ee0b9454a485e4662a8/raw/ef461672f0b869724215352866cd7137e973ce12/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

File: src/uct/tcp/tcp.h (hunk 1 above)
Comment (hiroyuki-sato) [reply src/uct/tcp/tcp.h]: fixed

File: src/ucs/async/async_fwd.h (hunk 3 above)
Comment (hiroyuki-sato) [reply src/ucs/async/async_fwd.h]: fixed

File: src/ucs/async/async.h (hunk 2 above)
Comment (hiroyuki-sato) [reply src/ucs/async/async.h]: fixed

**Comment by hiroyuki-sato** [comment]
Hello, @dmitrygx. Thank you for your review. I fixed, and force pushed. Please take a look when you get a chance.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/11342/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 33 workers (click for details)</summary>

**Note:** the logs will be deleted after 26-Jul-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/9bb6703908b2c9b30345c1cfc97a66560b74d296/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/178d04570455de658cc00bc6332d2cce7d4d4d0f/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/f4a9f7cc63d29a1d24981f5cad5b561f6d534b53/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/2f3e1fe43565b6a356f1be66b9b15b5cf7239b45/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/100d4294369502999b319d2f531ca933295991fc/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/21117a673a8a03ae5cae868993497ea58ed9983b/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/86436ad54e1e8dee684acca657ba4e15a6107c40/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/6259864ae32cab5debdc5a279b1a348d3ea638d1/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/435214f5d7b772d6424a0cf735b5def07c754ec2/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/42a389ab50cb78ed6d12f4c57aa8cd9e23553e0d/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/4a1529c2755f1da185f1b0e2722c0f764805c176/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/7dcdcdce3f50cbba8d284ed9d88a188ef82bec22/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/30f557b6f200bb5586d60c27819b40573c7f460e/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/93ad97270189fcdf3ca41dc5e8bce291eda4bc87/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/348a907608292456fee5b731f92321a88ce32285/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/8b788466f43c8de1519964061027de40c7dceea9/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/a9f9c71f83e19e5293f57922a6decbec09f7d109/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/4b30a9e414f30352cf628725452783ea37b934e0/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/b6b5c5060cbfb267646c31cb2586c79267cb1d61/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/535027167142ea7808ac7c68cfff710111091904/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/b0db42d21d421225393d5a3857f1f7b6034bfb8e/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/cc20c154e0eec9236e7c2a960f5e24649a5a7e77/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/5d2065e09f29c17c55a3471446cd8de70f46a031/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/be03c05b8057b7a48adb5df6c5c55333d2c8e6b4/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/a19eac0333bb695599c252d49515d7bc51158547/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W0](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/0a862c9f621f8579575b8d430c5e9acc46891159/hpc-test-node-upstream_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W1](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/f693a5aefbd01382e3a2d96d32b33e2fbccf5a3a/hpc-test-node-upstream_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W2](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/97df972fb477cf44e72bed6f8d7abe7b16933e8b/hpc-test-node-upstream_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W3](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/60061ccd2f52abfde9be948724b19cb07f17d667/hpc-test-node-upstream_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/9f6329bd0d9a9eba45095c40291fb0fa1dba817d/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/6d39ad07285265687b2ee37d7ba1d75af8f088e9/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/35cbcd348b656fe0c8cd3e3844b2bc984334b919/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/d12f33e220c1fa806583009a5965bbee/raw/02972e54b681ae93850bd95387a068d37a61321c/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by shamisp** [comment]
@tonycurtis can you please take a look as well

**Comment by dmitrygx** [comment]
Hello @hiroyuki-sato

Thank you!

**Comment by tonycurtis** [comment]
> async.h:56:               Either to use signals or polling threads to wait.
> 

Indicates whether to use signals or polling threads for waiting.

> async_fwd.h:109

ditto

> tcp.h:20

to wait on?


Tony

**Pull request merged**